src/components/modals/StepOneRegistration.tsx

```diff
@@ -0,0 +1,55 @@
+import Button from "../commons/Button";
+import { Imodal } from "../../types/types";
+import registerHandler from "../../actions/registerHandler";
+
+const StepOneRegistration: React.FC<Imodal> = ({
+  data,
```

Comment:
If it's possible, avoid using the word `data`. Everything's data, let's be more descriptive.

Reply:
Fixed in commit d5dd696371cd44e45af55bcbabd6811af380cde2.